src/sim_engine.rs

```diff
@@ -6,15 +6,21 @@ use blockdev::BlockDevs;
 
 use engine::Engine;
 
-use pool::StratisPool;
-
+use std::fmt::Debug;
 use std::path::Path;
 use std::collections::BTreeMap;
 use std::iter::FromIterator;
 
 use types::StratisResult;
 
 
+pub trait StratisPool: Debug {
```

Comment:
StratisPool is a trait that all Engine implementations will be using. Therefore it would be better to put it in engine.rs, because sim_engine.rs should just have the specific code and data types associated with its implementation of the Engine and StratisPool traits.